packages/wrangler/src/config/patch-config.ts

```diff
@@ -0,0 +1,63 @@
+import TOML from "@iarna/toml";
+import { applyEdits, format, parse as JSONCParse, modify } from "jsonc-parser";
+import { parseTOML, readFileSync } from "../parse";
+import type { RawConfig } from "./config";
+import type { JSONPath } from "jsonc-parser";
+
+export const experimental_patchConfig = (
+	configPath: string | undefined,
```

Comment:
Why `configPath` could be `undefined`?

Reply:
readRawConfig, resolveWranglerConfigPath all return a configPath as `string | undefined` for when there isn't one, so I preferred to handle it inside experimental_patchConfig but I don't feel strongly about it.

@penalosa the spec has this function taking `configPath?: string, config: RawConfig` - is there any particular reason for configPath being optional?